src/site/fml/faq.fml

```diff
@@ -720,7 +720,11 @@ public void testIndexOutOfBoundsExceptionNotRaised()
       For details on how to use it,
       read <a
       href="http://www.onjava.com/pub/a/onjava/2003/11/12/reflection.html">this
-      article</a>.
+      article</a>.<br>
```

Comment:
I think the `<br>` tag is what is breaking the build. Instead, use `<p>` and `</p>` around paragraphs.